<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActiveCodeRenewItem</class>
 <widget class="QWidget" name="ActiveCodeRenewItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>28</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>24</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>24</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"> QCheckBox {
     spacing: 5px;
	background:transparent;
 }

 QCheckBox::indicator {
     width: 16px;
     height: 16px;
 }

 QCheckBox::indicator:unchecked {
     image: url(:/login/resource/login/option_normal.png);
 }

 QCheckBox::indicator:unchecked:hover {
     image: url(:/login/resource/login/option_normal.png);
 }

 QCheckBox::indicator:unchecked:pressed {
     image: url(:/login/resource/login/option_normal.png);
 }

 QCheckBox::indicator:checked {
     image: url(:/login/resource/login/option_select.png);
 }

 QCheckBox::indicator:checked:hover {
     image: url(:/login/resource/login/option_select.png);
 }

 QCheckBox::indicator:checked:pressed {
     image: url(:/login/resource/login/option_select.png);
 }
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelRenewActiveCode">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;background:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelStatus">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;background:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolBtnDel">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;background:transparent;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/main/resource/main/deleteActiveItem.png</normaloff>:/main/resource/main/deleteActiveItem.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
